openshift-201/resource-mgmt.md

```diff
@@ -211,50 +216,169 @@ Try adjusting the limits and requests on our web server pod and running the load
+
+Important: 
+
+If a quota that restricts usage of compute resources for a project is set, then OpenShift refuses to create pods that do not specify resource requests or resource limits for that compute resource. To use most templates and builders with a project restricted by quotas, the project must also contain a limit range resource that specifies default values for container resource requests.
+
+**Quota scopes**
+
+Each quota can have an associated set of scopes. A quota only measures usage for a resource if it matches the intersection of enumerated scopes.
+
+Adding a scope to a quota restricts the set of resources to which that quota can apply. Specifying a resource outside of the allowed set results in a validation error.
+
+| Scope | Description|
+|---|---|
+|Terminating | Match pods where spec.activeDeadlineSeconds >= 0.|
+| NotTerminating | Match pods where spec.activeDeadlineSeconds is nil.|
+| BestEffort | Match pods that have best effort quality of service for either cpu or memory.|
+| NotBestEffort| Match pods that do not have best effort quality of service for cpu and memory.|
+
+You'll notice there are a few quotas within the BC Gov OpenShift projects. Have a look at quota details and see how much has been consumed.
+## Limit Ranges
 
-A limit range resource can also define default, minimum, and maximum values for the storage capacity requested by an image, image stream, or persistent volume claim. If a resource that is added to a project does not provide a compute resource request, then it takes the default value provided by the limit ranges for the project. If a new resource provides compute resource requests or limits that are smaller than the minimum specified by the project limit ranges, then the resource is not created. Similarly, if a new resource provides compute resource requests or limits that are higher than the maximum specified by the project limit ranges, then the resource is not created.
+A LimitRange resource, also called a limit, defines the default, minimum, and maximum values for compute resource requests, and the limits for a single pod or container defined inside the project. 
+
+To understand the difference between a limit range and a resource quota, consider that a limit range defines valid ranges and default values for a single container, and a resource quota defines only top values for the sum of all pods in a project. 
+
+A cluster administrator concerned about resource usage in an OpenShift cluster usually defines both limits and quotas for a project.
+
+A limit range resource can also define default, minimum, and maximum values for the storage capacity requested by an image, image stream, or persistent volume claim. 
```

Comment:
How do you set defaults for storage capacity for Images?

Reply:
Whoop. Typo has been fixed now.